src/mame/layout/enmirage.lay

```diff
@@ -0,0 +1,210 @@
+	<element name="lcd_background"><rect><color red="0.360" green="0.274" blue="0.180" /></rect></element>
+
+	<!-- Labels -->
+	<element name="label_1"> <text string="1"> <color red="1" green="1" blue="1" /></text></element>
+	<element name="label_2"> <text string="2"> <color red="1" green="1" blue="1" /></text></element>
+	<element name="label_3"> <text string="3"> <color red="1" green="1" blue="1" /></text></element>
+	<element name="label_Rec"> <text string="Rec"> <color red="1" green="1" blue="1" /></text></element>
+	<element name="label_Upper"> <text string="Upper"> <color red="1" green="1" blue="1" /></text></element>
+	<element name="label_4"> <text string="4"> <color red="1" green="1" blue="1" /></text></element>
+	<element name="label_5"> <text string="5"> <color red="1" green="1" blue="1" /></text></element>
+	<element name="label_6"> <text string="6"> <color red="1" green="1" blue="1" /></text></element>
+	<element name="label_Play"> <text string="Play"> <color red="1" green="1" blue="1" /></text></element>
+	<element name="label_Lower"> <text string="Lower"> <color red="1" green="1" blue="1" /></text></element>
+	<element name="label_Param"> <text string="Param"> <color red="1" green="1" blue="1" /></text></element>
+	<element name="label_Value"> <text string="Value"> <color red="1" green="1" blue="1" /></text></element>
+	<element name="label_7"> <text string="7"> <color red="1" green="1" blue="1" /></text></element>
+	<element name="label_8"> <text string="8"> <color red="1" green="1" blue="1" /></text></element>
+	<element name="label_9"> <text string="9"> <color red="1" green="1" blue="1" /></text></element>
+	<element name="label_Load"> <text string="Load"> <color red="1" green="1" blue="1" /></text></element>
+	<element name="label_Off"> <text string="Off/&#9662;"> <color red="1" green="1" blue="1" /></text></element>
+	<element name="label_On"> <text string="On/&#9650;"> <color red="1" green="1" blue="1" /></text></element>
+	<element name="label_Cancel"> <text string="Cancel"> <color red="1" green="1" blue="1" /></text></element>
+	<element name="label_0"> <text string="0/Prog"> <color red="1" green="1" blue="1" /></text></element>
+	<element name="label_Enter"> <text string="Enter"> <color red="1" green="1" blue="1" /></text></element>
+	<element name="label_Save"> <text string="Save"> <color red="1" green="1" blue="1" /></text></element>
+	<element name="label_CONTROL"> <text string="CONTROL"> <color red="1" green="1" blue="1" /></text></element>
+	<element name="label_SELECT"> <text string="SELECT"> <color red="1" green="1" blue="1" /></text></element>
+	<element name="label_SEQ"> <text string="SEQ"> <color red="1" green="1" blue="1" /></text></element>
+	<element name="label_SAMPLE"> <text string="SAMPLE"> <color red="1" green="1" blue="1" /></text></element>
+	<element name="label_LOAD"> <text string="LOAD"> <color red="1" green="1" blue="1" /></text></element>
```

Comment:
You don’t need to put explicit colour for white – it’s implied, since it’s the unity matrix.